srcpkgs/lumina/patches/checkpass.patch

```diff
@@ -0,0 +1,19 @@
+--- a/src-qt5/core/lumina-checkpass/lumina-checkpass.pro
++++ b/src-qt5/core/lumina-checkpass/lumina-checkpass.pro
+@@ -6,13 +6,13 @@ QT =
+ CONFIG += console
+ 
+ TARGET = lumina-checkpass
+-target.path = $$DESTDIR$${PREFIX}/sbin
++target.path = $$DESTDIR$${PREFIX}/bin
+ 
+ LIBS     += -lpam
+ 
+ SOURCES += main.c
+ 
+-perms.path = $$DESTDIR$${PREFIX}/sbin
+-perms.extra = "chmod 4555 $$DESTDIR$${PREFIX}/sbin/lumina-checkpass"
++perms.path = $$DESTDIR$${PREFIX}/bin
++perms.extra = "chmod 4555 $(INSTALL_ROOT)$$DESTDIR$${PREFIX}/bin/lumina-checkpass"
```

Comment:
Please add an explanation for what the patch does on top of the file

Reply:
The lumina-checkpass binary, which I understand is to be disabled in release 1.6.2 according to comments on the project site (https://github.com/lumina-desktop/lumina/issues/774), was set to install to a hardwired "sbin" location, but that generates a conflict with the pkg-src sanity/lint tests. This patch just forces the build to install to a "bin" path resulting in the pkg-src checks being passed. The need for this patch should disappear with 1.6.2 if the 774 issue cited is to be believed.

Reply:
patch updated with brief comment